_includes/_acquire_details.htm

```diff
@@ -1,14 +1,37 @@
 <section class="acquire-details grc-bgcolor" data-background-color="black">
     <div class="container">
         <div class="row">
-            <div class="col-xs-12 col-sm-12 col-md-12 col-lg-12">
-                <h3>How to buy Gridcoin?</h3>
-                <p>On most centralized exchanges you need to first <a href="https://www.bitcoin.com/buy-bitcoin">buy Bitcoin</a> in order to buy Gridcoin, except from <a href="https://flyp.me">Flyp.me</a> where you can exchange several cryptocurrencies for Gridcoin.</p>
+            <div class="col-xs-12 col-sm-12 col-md-12 col-lg-12 text-center">
+                <h3 class="text-bold">Ways to Get Gridcoin</h3>
+                <p>If you don't want to buy any Gridcoin, you can get some through the faucets or <a href="/Guides/pool.htm">through pool mining</a>. On most centralized exchanges you need to first <a href="https://www.bitcoin.com/buy-bitcoin">buy Bitcoin</a> in order to buy Gridcoin, except from <a href="https://flyp.me">flyp.me</a> where you can use other cryptocurrencies to get Gridcoin.</p>
```

Comment:
Could we split these sentences perhaps. Add a <br> just after the pool mining link, makes it flow better and easier to read.

Reply:
Not sure that looks better 
![image](https://user-images.githubusercontent.com/30132912/57985599-28a26780-7a38-11e9-8ba9-3bbc807796f3.png)

Reply:
My reasoning is: the first sentence is about not buying GRC and the second is about buying. 
The two sentences are about two differing topics so when one comes straight after the other without a line break it just feels and reads weirdly to me which is why I prefer the line break.

Reply:
Yeah, I guess. I'll add the commit